vaadin-text-field-flow-parent/vaadin-text-field-flow/src/main/java/com/vaadin/flow/component/textfield/AbstractNumberField.java

```diff
@@ -16,6 +16,7 @@
 
 package com.vaadin.flow.component.textfield;
 
+import com.vaadin.flow.component.HasClearButton;
```

Comment:
nit: let's place this import together with the rest of `com.vaadin.flow.component` imports.

Reply:
Done